src/v2/api/controllers/UserController.ts

```diff
@@ -668,7 +771,7 @@ export class UserController {
   @UseGuards(TelegramGuard)
   @Get('/telegramUser/:telegramId')
   async getUserByTelegramId (
-      @Param('telegramId', UserByTelegramIdPipe) telegramId: bigint,
+    @Param('telegramId', UserByTelegramIdPipe) telegramId: bigint,
   ) {
     const student = await this.userService.getUserByTelegramId(telegramId);
     return this.studentMapper.getStudent(student);
```

Comment:
Line 780, move Access to ApiEndpoint, add ApiEndpoint with summary. And you should do it to all Access: delete them and move permission to ApiEndpoint

Reply:
Not my endpoint